### PR TITLE
ChartView: modify UI of placeholder for repositories with no author commits

### PR DESCRIPTION
**Pull request by jamessspanggg**
```
Within ChartView, ramp charts with error messages are blended together 
with the other ramp charts without error messages as they both have 
similar UI.

Let's modify the UI of ramp charts with error messages to use red color 
for their info message font to distinguish itself from other ramp 
charts without error messages.
```

**Updated UI**

group by repo:
Screen Shot 2019-07-05 at 11 34 54 AM

group by author:
Screen Shot 2019-07-05 at 11 35 05 AM

group by none:
Screen Shot 2019-07-05 at 11 35 15 AM

### CODE REVIEW

**Comment by fzdy1914** [comment]
@damithc Your opinion of the UI change?

**Comment by damithc** [comment]
> @damithc Your opinion of the UI change?

I'm OK with it 👍

File: frontend/src/index.pug [hunk 1]
```diff
@@ -131,7 +131,10 @@ html
           .summary-charts(v-for="repo of filteredRepos")
             .summary-charts__title(v-if="filterGroupSelection === 'groupByRepos'") {{ repo[0].repoName }}&nbsp;&nbsp;
               .summary-charts__title--contribution(title="Total contribution of group") [{{ getGroupTotalContribution(repo) }} lines]
-            .summary-charts__title(v-if="filterGroupSelection === 'groupByAuthors'") {{ repo[0].displayName }} ({{ repo[0].name }})&nbsp;&nbsp;
+            .summary-charts__title(
+                v-if="filterGroupSelection === 'groupByAuthors'",
+                v-bind:class="{ red: repo[0].name === '-' }"
```
Comment (fzdy1914) [inline frontend/src/index.pug]: The class name `red` is a bit weird. But I cannot find a better one myself.

Comment (jamessspanggg) [reply frontend/src/index.pug]: how about `warning`?

Comment (fzdy1914) [reply frontend/src/index.pug]: Can you check on the `.warn` class of CSS? `.warning` is also better than `red`.

**Pull request merged**